torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1975,16 +1969,13 @@ at::Tensor XLANativeFunctions::mean(const at::Tensor& self,
       /*keep_reduced_dimensions=*/false, dtype));
 }
 
-at::Tensor XLANativeFunctions::mean(const at::Tensor& self,
-                                    at::OptionalIntArrayRef dim, bool keepdim,
+at::Tensor XLANativeFunctions::mean(const at::Tensor& self, at::IntArrayRef dim,
```

Comment:
I don't think these changes are from you, rebase should solve the issue.

Reply:
Done